contracts/staking/stakeManager/StakeManagerStorage.sol

```diff
@@ -82,4 +93,7 @@ contract StakeManagerStorage is GovernanceLockable, RootChainable {
     mapping(uint256 => uint256) public latestSignerUpdateEpoch;
 
     uint256 public totalHeimdallFee;
+
+    uint256 public rewardPerStake;
+    address auctionImplementation;
```

Comment:
Shouldn't this mess with `StakeManagerStorage, Initializable` order for `StakeManager` since `Initializable` also has storage?

Reply:
It is going to mess, yes, I am looking into it.